server/utils/schedule.js

```diff
@@ -0,0 +1,26 @@
+const schedule = require('node-schedule')
+const Similarity = require('./similarity')
+const addedTrack = require('../models/addedTrack')
+
+/* job that updates all similarity scores in database every 5th minute */
+schedule.scheduleJob('*/5 * * * *', async () => {
+  await Similarity.updateSimilarityScores()
+})
+
+/* job that removes expired track records at the start of every hour */
+schedule.scheduleJob('0 * * * *', async () => {
+  const tracks = await addedTrack.find()
+
+  /* delete expired tracks */
```

Comment:
The more reusable way is to have this logic in a method and just have your scheduler call methods. This way you can have multiple scheduling mechanisms. It also makes it easier to test.